teamA/jeongDoYoung/app.js

```diff
@@ -0,0 +1,33 @@
+//built-in package
+const http = require("http");
+//3rd-party package
+const express = require("express");
+const morgan = require("morgan");
+const cors=require("cors");
+const dotenv = require("dotenv").config();
```

Comment:
- dotenv를 아래 코드로 수정해서 최상단에 작성해주세요 !
```suggestion
require("dotenv").config();
```